packages/scalable-data-sync/package.json

```diff
@@ -0,0 +1,84 @@
+{
+    "name": "@waku/scalable-data-sync",
+    "version": "0.0.1",
+    "description": "Scalable Data Sync implementation for the browser",
```

Comment:
nit: we can give a link to RFC here and in `Readme`

Reply:
There is a link in the readme to https://github.com/vacp2p/rfc-index/blob/main/vac/raw/sds.md

added to description here as well